source/_data/glossary.yml

```diff
@@ -275,6 +275,15 @@
     Services are called to perform actions.
   link: /docs/scripts/service-calls/
 
+- term: State
+  definition: |-
+    State is a data container holding the current state of an entity. E.g. if a light is on or off, the current temperature, or the RGB value of a light. The data type of state is `String`. States have an entity id, a state value, and 2 timestamps: `last_updated` and `last_changed`. Apart from the actual state value, a state can have attributes and context. 
```

Comment:
The "State is a data container holding" a bunch of things, v.s. "The data type of state is `String`" is conflicting in this sentence.

I think the main issue is the confusion with a full-state object in this case? From a state machine perspective, the state objects are the entities @ Runtime. I don't think for the end-user part, we need to maintain that difference.

From a user perspective, I think we can refer to the state object as "the entity".

The state of an entity is, indeed, always a string.

Reply:
Ah, yes. I did confuse it with the state object. Let me try this again.